docs/advanced/input_files/input-main.md

```diff
@@ -147,6 +147,7 @@
     - [out\_mat\_t](#out_mat_t)
     - [out\_mat\_dh](#out_mat_dh)
     - [out\_mat\_xc](#out_mat_xc)
+    - [out\_eband\_separate\_term](#out_eband_separate_term)
```

Comment:
the name is too long

Reply:
"separate_term" is replaced by "terms" in the newest commit.